### PR TITLE
feat: added card component

### PR DESCRIPTION
**Pull request by sks-21**
Added **card** component
Includes 

- Vertical Card
- Card with badges
- Card with dismiss
- Card with text overlay
- Text only card
- Card with shadow
- Horizontal card

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+/* || Vertical card  */
+.ver-card {
+  border: 2px solid var(--color-offwhite);
+  border-radius: 10px;
+  margin: 1rem;
+  font-size: 1.6rem;
+  padding: 0.8rem;
+  width: 25rem;
+
+  position: relative;
```
Comment (barbaricankit) [inline components/card/card.css]: can name it vertical card itself

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+}
+
+.content-img img {
+  width: 100%;
+  height: 65%;
+  border-radius: 5px;
+}
```
Comment (barbaricankit) [inline components/card/card.css]: could have given the class name to img tag

File: components/card/card.css [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+.overlay-btns button {
+  background: var(--color-gray-light);
+  border-radius: 50%;
+  margin: 4px 2px 0 0;
+
+  padding: 6px;
+  font-size: 2rem;
+}
```
Comment (barbaricankit) [inline components/card/card.css]: similarly for buttons use the class name in the tag itself for which you want to design

File: components/card/card.css [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+
+.card-title {
+  color: var(--color-gray);
+}
+
+.card-desc {
+  color: var(--color-black);
+  font-size: bold;
+}
```
Comment (barbaricankit) [inline components/card/card.css:64]: for variables give primary color or secondary color

File: components/card/card.css [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+
+.card-price-mrp {
+  text-decoration: line-through;
+  color: var(--color-gray);
+}
+
+.card-price-disc {
+  color: var(--color-green);
+}
+
```
Comment (barbaricankit) [inline components/card/card.css]: when you will be building a card on a different project, there you will not be able to use this class, so give generic names.
Like card-header/title, card-body, card-sub-body.

File: components/card/card.css [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+/* || Card with badges */
+
+.overlay-btns button {
+  margin-left: auto;
+}
+
+.overlay-btns span {
+  /* margin: 0rem; */
```
Comment (barbaricankit) [inline components/card/card.css]: remove comments

File: components/card/card.css [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+  height: 3rem;
+}
+
+/* || Text only Cards  */
+
+.txt-card {
+  width: 35rem;
+}
+
+.txt-card-headings {
```
Comment (barbaricankit) [inline components/card/card.css]: use singular name txt-card-heading

File: components/card/card.css [hunk 8]
```diff
@@ -0,0 +1,233 @@
...
+}
+
+.txt-card-content {
+  margin-top: 8px;
+}
+
+.txt-card-btns {
+  align-items: center;
+  margin-top: 8px;
+}
```
Comment (barbaricankit) [inline components/card/card.css]: similarly btn

**Review by Manju-sree — APPROVED** [review]
looks good to merge

**Review by Namrata-J — APPROVED** [review]

File: components/card/card.css [hunk 9]
```diff
@@ -0,0 +1,233 @@
...
+  height: 95%;
+}
+
+.overlay-full-btns {
+  height: inherit;
+
+  justify-content: center;
+  align-items: center;
+}
+
```
Comment (barbaricankit) [inline components/card/card.css]: use prettier

File: components/card/card.css [hunk 10]
```diff
@@ -0,0 +1,233 @@
...
+/* || Horizontal card  */
+.horz-card {
+  border: 2px solid var(--color-offwhite);
+  border-radius: 10px;
+  margin: 1rem;
+  font-size: 1.6rem;
+  padding: 0.8rem;
+  height: 25rem;
+  width: 45rem;
+  position: relative;
```
Comment (barbaricankit) [inline components/card/card.css]: can use the horizontal class name.

**Review by barbaricankit — COMMENTED** [review]
Please look into my comments.

File: components/card/card.css (hunk 4 above)
Comment (sks-21) [reply components/card/card.css:64]: I want it to look this particular way. I have added other utility classes by which user can change to the desired color

**Pull request merged**